### ISSUE TITLE
How many num_steps should we set to config file to retrain Open Images?

### ISSUE THREAD

**Issue by AI-LastWish** [tensorflow/models#6506]
I'm going to retrain faster_rcnn_nas_lowproposals_coco model with Open Images dataset and I wonder which number I should set to num_steps in file faster_rcnn_nas_lowproposals_coco.config? 

The default value is 200000, but I'm afraid with 1.7 Million images, it may take very long time to finish.

Anyone can suggest me a number?

System information
What is the top-level directory of the model you are using:(I don't really understand the meaning of "top-level directory of the model")
I just followed https://github.com/EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10

I used faster_rcnn_nas_lowproposals_coco downloaded from
https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md

OS Platform and Distribution:
Ubuntu 16.04
TensorFlow installed from:
GPU version of Tensorflow
TensorFlow version:
v1.12
Bazel version:
NA: don't know what is Bazel
CUDA/cuDNN version:
NA: don't know, but it's not the related to the problem
GPU model and memory:
Titan V
Exact command to reproduce:
NA

**Comment by rootkitchao** [comment]
The number of steps is not epoch.The number of training steps depends on the learning rate setting and the batch size.

**Comment by AI-LastWish** [comment]
> train_config: {
  batch_size: 1
  optimizer {
    momentum_optimizer: {
      learning_rate: {
        manual_step_learning_rate {
          initial_learning_rate: 0.0003
          schedule {
            step: 900000
            learning_rate: .00003
          }
          schedule {
            step: 1200000
            learning_rate: .000003
          }
        }
      }

**Comment by rootkitchao** [comment]
I am sorry that I have not tried to train the model on OID.If you download an OID model from the model zoo, you can find a model for training from scratch.Note that the hyperparameters recorded in pipeline.config may be for multi-GPU training.
If you plan to adjust the parameters yourself, it is recommended to try the initial learning rate from [0.0001, 0.001] and observe the convergence to manually reduce the learning rate.

**Comment by paviddavid** [comment]
@AI-LastWish I am thinking about the same issue with another data set. How many num_steps have you used for the fine-tuning? Just to have a reference point..
